packages/vite/src/node/plugins/importMetaGlob.ts

```diff
@@ -437,6 +458,11 @@ export async function transformGlobImport(
             let importPath = paths.importPath
             let importQuery = options.query ?? ''
 
+            if (onlyKeys) {
+              objectProps.push(`${JSON.stringify(filePath)}: ''`)
```

Comment:
I just noticed that we can trim one character for each entry with this.
```suggestion
              objectProps.push(`${JSON.stringify(filePath)}: 0`)
```

Reply:
Thanks!